components/Habit.tsx

```diff
@@ -89,25 +111,23 @@ const Habit = ({ entries, habit, streak }: HabitProps) => {
         <span>{habit.name}</span>
         <span css={styles.deadline}>
           {completionPercentage < 100
-            ? t('common:dayLeft', { quantity: '2' })
+            ? t('common:daysLeft', { quantity: '2' })
```

Comment:
`quantity: 2`?

Reply:
So it'll say <quantity> days left. This needs to be calculated at some point in another PR.